doc/classes/DisplayServer.xml

```diff
@@ -560,6 +560,16 @@
 			<description>
 			</description>
 		</method>
+		<method name="screen_get_refresh_rate" qualifiers="const">
+			<return type="float">
+			</return>
+			<argument index="0" name="screen" type="int" default="-1">
+			</argument>
+			<description>
+				Return's the screen refresh rate.
+				[b]Note:[/b] On macOS and iPhone, this will always be 60.
```

Comment:
I would use `this is` instead of `this will always be` since it could change in the future.

Reply:
This line won't be in the final merged version of this PR, since I'm adding support for getting the refresh rate through AppKit

Reply:
I would also replace "iPhone" with "iOS", since Godot also supports iPadOS and (soon) Apple TV.